packages/app-elements/src/ui/forms/InputSelect/AsyncComponent.tsx

```diff
@@ -13,6 +14,10 @@ interface AsyncSelectComponentProps
     'label' | 'hint'
   > {
   styles: StylesConfig<InputSelectValue>
+  /**
+   * Message shown if the select is `async` and `initialValues` prop is set to []
+   */
+  asyncEmptySearchMessage?: string
```

Comment:
We decided with Simone to hide the `div` when focus is on the input and we still need to start typing

Reply:
Done! It was enough to return `null` in required conditions :)